features/admin.logs.v1/pages/audit-logs-page.tsx

```diff
@@ -51,7 +52,7 @@ type AuditPagePropsInterface = IdentifiableComponentInterface;
  *
  * @returns Logs Page {@link React.ReactElement}
  */
-export const AuditLogsPage = (props: AuditPagePropsInterface) : ReactElement => {
+const AuditLogsPage = (props: AuditPagePropsInterface) : ReactElement => {
```

Comment:
Have we removed `export` because it is not being used?

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/6773/commits/b900c355f3611c11828b641120f7afe5ac85d382